packages/webapp/src/pages/Connection/Authorization.tsx

```diff
@@ -46,6 +49,25 @@ export default function Authorization(props: AuthorizationProps) {
                 )}
             </div>
             <div className="flex">
+                <div className="flex flex-col w-1/2">
```

Comment:
Do we want to show these fields here if not present? Most connections won't have it and it takes up a lot of space with no values.